src/main/java/com/github/cameltooling/lsp/internal/parser/ParserFileHelper.java

```diff
@@ -26,7 +26,8 @@
 
 public abstract class ParserFileHelper {
 	
-	protected static final List<String> CAMEL_POSSIBLE_TYPES = Arrays.asList("to", "from");
+	protected static final List<String> CAMEL_POSSIBLE_TYPES = Arrays.asList("to", "from", "endpoint");
+	protected static final List<Character> POSSIBLE_URI_CLOSURE_CHARS = Arrays.asList('\"', '\'');
```

Comment:
shouldn't be better placed in the ParserXMLFileHelper? The Java xml parser is not using it right?

Reply:
right